contact.html

```diff
@@ -0,0 +1,613 @@
+                                                    </svg>
+                                                  </span>
+                                                </a>
+                                              </div>
+                                              <div class="button aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <a id="button-229994a626" class="cmp-button" href="https://pinterest.com">
+                                                  <span class="cmp-button__icon cmp-button__icon--icon-pinterest" aria-hidden="true">
+                                                    <svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                                        <circle cx="17.623" cy="17.623" r="17.123" stroke="#FFCF07"/>
+                                                        <g clip-path="url(#clip0_447_2113)">
+                                                          <path d="M16.58 12H12V16.3027V20.6054V24.7151H16.58V20.6054H21.0867V16.3027H25.5346V12H16.58Z" fill="#383F43"/>
+                                                        </g>
+                                                        <defs><clipPath id="clip0_447_2113"><rect width="13.5346" height="12.6323" fill="white" transform="translate(12 12)"/></clipPath></defs>
+                                                      </svg>
+                                                  </span>
+                                                </a>
+                                              </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="container responsivegrid contact-form aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div id="contact-container" class="cmp-container">
+                          <div class="text formtext aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="text-cdbe0aaaa2" class="cmp-text">
+                                <p>This is a safe place to learn, grow, and ask quesitons. We will protect your identity and never use this information for marketing purposes.</p>
+                            </div>
+                          </div>
+                          <div class="container responsivegrid form-container aem-GridColumn aem-GridColumn--default--12 layout-grid">
```

Comment:
@Prystal254 This whole form needs to be redone using AEM's form [container](https://www.aemcomponents.dev/content/core-components-examples/library/core-content/form-container.html), [text](https://www.aemcomponents.dev/content/core-components-examples/library/core-form/form-text.html), [options](https://www.aemcomponents.dev/content/core-components-examples/library/core-form/form-options.html), and [button](https://www.aemcomponents.dev/content/core-components-examples/library/core-form/form-button.html) components.